Sources/muterCore/CLICommands/RunCommand/Steps/DiscoverMutationPoints.swift

```diff
@@ -29,7 +29,7 @@ private extension DiscoverMutationPoints {
         let mutationPoints: [MutationPoint] = filePaths.accumulate(into: []) { alreadyDiscoveredMutationPoints, path in
             
             guard pathContainsDotSwift(path),
-                let source = sourceCode(fromFileAt: path) else {
+                  let source = sourceCode(fromFileAt: path)?.code else {
```

Comment:
```suggestion
                let source = sourceCode(fromFileAt: path)?.code else {
```

I think? Just looking at GitHub; I didn't check in Xcode

Reply:
huh, actually, Xcode wants to indent the _next_ line weirdly. What do you think about this?

<img width="444" alt="image" src="https://user-images.githubusercontent.com/464574/92842643-d23a0480-f3b1-11ea-9614-616079641aee.png">

Reply:
I agree, it's weird...I'm not sure if worth fighting against Xcode without any other tool (like SwiftLint)